airflow/contrib/hooks/gcs_hook.py

```diff
@@ -577,14 +577,8 @@ def insert_bucket_acl(self, bucket, entity, role, user_project):
         :param bucket: Name of a bucket.
         :type bucket: str
         :param entity: The entity holding the permission, in one of the following forms:
-        - user-userId
-        - user-email
-        - group-groupId
-        - group-email
-        - domain-domain
-        - project-team-projectId
-        - allUsers
-        - allAuthenticatedUsers
+            user-userId, user-email, group-groupId, group-email, domain-domain,
```

Comment:
nit: link to docs (is https://cloud.google.com/storage/docs/access-control/lists#scopes sufficient? is there a better doc?)